ui/Screen/Project/Source/Commits.svelte

```diff
@@ -51,26 +52,23 @@
 </style>
 
 <div class="commits-page" data-cy="commits-page">
-  <Remote {store} let:data={commits}>
-    {#each commits.history as history}
+  <Remote {store} let:data={{ history }}>
+    {#each history.history as group (group.time)}
       <div class="commit-group">
         <header>
-          <p>{formatTime(history.time * 1000)}</p>
+          <p>{formatTime(group.time * 1000)}</p>
```

Comment:
What's this `* 1000` logic?

Reply:
Because every language defines epoch as seconds while Brendan Eich deemed it fitting to add some millis...

Reply:
Madness. I see we do it a lot, would it be worth having a helper?